test/smooth_test.cpp

```diff
@@ -111,8 +109,7 @@ TEST(Smooth, ToLength) {
       CrossSection::Circle(10, 10).Translate({10, 0}).ToPolygons(), 2, 0, 0,
       {0, 0});
   cone += cone.Scale({1, 1, -5});
-  Manifold smooth = Manifold::Smooth(cone.GetMesh());
-  smooth = smooth.RefineToLength(0.1);
+  Manifold smooth = cone.AsOriginal().SmoothOut(180).RefineToLength(0.1);
```

Comment:
Okay, I fixed it with `AsOriginal` and I think I understand why - our old Mesh didn't keep track of IDs, which allowed those faces to be collapsed, but MeshGL keeps more info. I think this is fine.